### ISSUE TITLE
Facing an issue while installing the basic installation using cloud formation template

### ISSUE THREAD

**Issue by bhargav-b** [FINRAOS/herd-mdl#84]
Hi team,

We are trying install the Finra Herd MDL using "Basin install" yaml file (installMDL.yaml) as per the instructions mentioned in https://finraos.github.io/herd-mdl/docs.html .
But facing the issue mentioned in attached screen shot. We have Poweruser privileges too.

Any suggestions would be greatly appreciated. If it is not the right forum, the could you please  let me know the right forum.

Thanks in advance.

Regards,
Bhargav
![MDLissue](https://user-images.githubusercontent.com/30067565/54690831-3b87e100-4b48-11e9-91a5-67e7d15a229e.jpg)

**Comment by rongwang0930** [comment]
can you take a look at this ticket to see if this works for you?
https://github.com/FINRAOS/herd-mdl/issues/82

**Comment by bhargav-b** [comment]
@rongwang0930 Sure will try and update. thank you :)

**Comment by bhargav-b** [comment]
Now it is working fine :)..
But one issue I am facing now. here it is. "ami-1853ac65 is invalid". Is this the ID of a Linux Image, if so I will update it according to my region.

**Comment by rongwang0930** [comment]
yes, it's the aws Linux image id,  please select one available in your region.

**Comment by bhargav-b** [comment]
okay. thank you. Will update you.

**Comment by bhargav-b** [comment]
I am able to resolve that issue. But Facing some other issue, while creating Web server. Please find the attached screenshot.
I have modified as per our earlier discussion.
1. I have set MetastorDBClass to - db.m5.large
2. Image Id - ami-0ad42f4f66f6c1cc9 - (ap-south-1 region)

Any suggestions please. Thanks in advance.

Regards,
Bhargav

![issue1](https://user-images.githubusercontent.com/30067565/54735716-c8c14900-4bcd-11e9-8b5a-ae53fcdcf703.PNG)
